addons/binding/org.openhab.binding.evohome/ESH-INF/thing/channels.xml

```diff
@@ -37,5 +37,11 @@
         <category>heating</category>
         <state min="4.5" max="30" step="0.1" pattern="%.1f °C" readOnly="false"/>
     </channel-type>
-
+    <channel-type id="setpointstatus">
+        <item-type>String</item-type>
+        <label>Setpoint Status</label>
+        <description>Thermostat setpoint status</description>
+        <category>heating</category>
+        <state readOnly="true"/>
```

Comment:
Did you want to make this readonly intentionally? I can see that this is a value one would like to change later? Or are you planning to change it once it is possible (in that case: kudos)